packages/lix-sdk/src/discussion/create-discussion.ts

```diff
@@ -3,28 +3,51 @@ import type { Lix } from "../lix/open-lix.js";
 
 /**
  * Creates a new discussion with the first comment.
+ * 
+ * Can be associated with a change set if provided.
  *
  * @example
  *   ```ts
+ *   // With a change set
  *   const changeSet = await createChangeSet({ lix, changes: ["change1", "change2"] });
- *   const discussion = await createDiscussion({ lix, changeSet, firstComment: { content: "first comment" } });
+ *   const discussion = await createDiscussion({ 
+ *     lix, 
+ *     firstComment: { content: "first comment" },
+ *     changeSet // optional
+ *   });
+ *   
+ *   // Without a change set (for inline annotations, etc.)
+ *   const discussion = await createDiscussion({ 
+ *     lix, 
+ *     firstComment: { content: "This is an inline annotation" }
+ *   });
  *   ```
  *
  * @returns the created discussion
  */
 export async function createDiscussion(args: {
 	lix: Pick<Lix, "db">;
-	changeSet: Pick<ChangeSet, "id">;
 	firstComment: Pick<Comment, "content">;
+	changeSet?: Pick<ChangeSet, "id">;
```

Comment:
let's remove the `changeSet` prop here and instead update `createChangeSet` to have a discussion prop